### ISSUE TITLE
Running Run-TestsInBcContainer with -testSuite defining a new test suite does not execute the tests

### ISSUE THREAD

**Issue by lvanvugt** [microsoft/navcontainerhelper#2087]
### Issue
When running `Run-TestsInBcContainer` with `-testSuite` defining a new test suite does not execute the tests, but only creates the new test suite.

**Scripts used to run the test in container container and cause the issue**
```
# container name and version
$containerName = 'd365bc18WA'

# set credentials for using with Windows
$login = 'xxx'
$password = 'yyy'
$securepassword = (ConvertTo-SecureString -String $password -AsPlainText -Force)
$credential = New-Object System.Management.Automation.PSCredential($login, $securepassword)

Run-TestsInBcContainer -containerName $containerName `
                       -credential $credential `
                       -extensionId 'e26890f8-fafe-49c6-8951-2c1457921f9b' `
                       -detailed `
                       -testSuite $containerName `
                       -testPage 130455 `
                       -testRunnerCodeunitId 130451
```

### Expected behavior
Tests are executed with the first call to `Run-TestsInBcContainer` that creates a new test suite.

**Comment by freddydk** [comment]
testSuite and extensionId are mutually exclusive.
extensionId will run all tests in an extension
testSuite will run all tests in a pre-defined test suite.

**Comment by lvanvugt** [comment]
OK, tested and works indeed when only using one of them. No way of having your script telling me that it's either one or the other?

**Comment by freddydk** [comment]
My script should tell me what parameters you submit? :-)
Not sure I understand your suggestion?

**Comment by lvanvugt** [comment]
Well, if these two parameters are mutually exclusive couldn't I be notified on that they exclude each other and that it doesn't makes sense to select both? This would have prevented this issue, right?

**Comment by freddydk** [comment]
Got it:-)

**Comment by lvanvugt** [comment]
Thanx

**Comment by freddydk** [comment]
Shipped in 2.0.16
